src/App.tsx

```diff
@@ -10,33 +11,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const onClickTab = (selectedTab: Tab) => setSelectedTabId(selectedTab.id);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        {' '}
+        {(tabs.find(tab => tab.id === selectedTabId))?.title}
```

Comment:
You can move this logic to outside function, e.g.

```
const selectedTabTitle = tabs
    .find(({ id }) => id === selectedTabId)?.title;
```

And then use it here,e.g.

```
<h1 className="title">
 {`Selected tab is ${selectedTabTitle}`}
</h1>
```

Reply:
I moved title to external function.